docs/extensions/creating-extensions.md

```diff
@@ -19,31 +18,57 @@ utility will create a folder with the recommended file structure to create an ex
 If you want to combine and share dependencies between one or more extensions, use the
 [bundle extension type](/extensions/bundles).
 
-### Extension Folder Structure
+## Secure Extensions
```

Comment:
Let's bump this whole thing to a marketplace section, and remove it from this PR. We'll push this PR sooner, and then make secure extensions (as well as docs around the marketplace) available when it's ready in preview.